go.mod

```diff
@@ -3,54 +3,54 @@ module sigs.k8s.io/aws-iam-authenticator
 go 1.16
 
 require (
-    github.com/aws/aws-sdk-go v1.43.28
```

Comment:
I assume the formatting change was just due to a new go version.  Would you mind reformatting in a separate commit from the version bump so its easier to tell that change is alone?

Reply:
The `go.sum` looks minimal. Is your editor adding tabs for spaces?

Reply:
Not sure what caused the formatting to change, I just ran `go get github.com/aws/aws-sdk-go@v1.44.107`. I fixed it